tests/adsb_test.py

```diff
@@ -0,0 +1,63 @@
+from pyModeS import adsb
```

Comment:
this must be relative import, because we are testing the current code, not the install package.